library/packages/src/modules/PackageSystem.rb

```diff
@@ -410,38 +410,35 @@ def CheckAndInstallPackages(packages)
     def CheckAndInstallPackagesInteractive(packages)
       packages = deep_copy(packages)
       success = CheckAndInstallPackages(packages)
-      if !success
-        if !LastOperationCanceled()
-          if Mode.commandline
-            # error report
-            Report.Error(_("Installing required packages failed."))
-          else
-            success = Popup.ContinueCancel(
-              # continue/cancel popup
-              _(
-                "Installing required packages failed. If you continue\n" \
-                  "without installing required packages,\n" \
-                  "YaST may not work properly.\n"
-              )
-            )
-          end
+      return success if success
```

Comment:
Well, yes, but I wanted to keep the original semantics just to not break something by mistake.